### PR TITLE
Desktop Icons and Workspaces updates

### PR DESCRIPTION
**Pull request by johnandmegh**
## Description

I tried to use some tips on Stack Overflow to "squash" the GitHub commits on my own this time...and somehow I think I made it even messier? (I don't think the end result is changed, but please let me know if I need to go back to the drawing board on how this is submitted)

Also, if this style of update is OK, I could probably tackle the rest of that page - I'm new all of Solus and Budgie, but it seems like a few different settings-related functions got moved from being accessed via Raven to being accessed via the Budgie Menu > Settings? Those would seem easy enough for me to grab new screenshots for and update wording.

Thanks,

Edited a few minutes later: sorry, meant to mention, this is related to #131

### Submitter Checklist

- [ ] Squashed commits with `git rebase -i` (if needed)

### CODE REVIEW

**Comment by davidjharder** [comment]
Git workflow stuff: 
- In this particular case, the mess is fine because I can squash it.
- Are you using the github desktop client, the command line, or something else?
- I would encourage you to experiment with grouping multiple changes onto a commit
- "squashing" commits is actually using the `rebase`  command to organize a pile of commits in a certain way. The most concise explanation I've found is here https://docs.github.com/en/get-started/using-git/about-git-rebase
- For a more general git resource try https://git-scm.com/book/en/v2

Style:
- The style seems fine. In my opinion It's more important to update these rather old pages
- If you *do* want to take a deep dive into style, we recently published a style guide. Note that we haven't used it much yet https://help.getsol.us/docs/user/contributing/style

I'll review the content of your changes some time later; I wanted to respond to your questions first.

**Comment by johnandmegh** [comment]
Thanks very much for the links, I will dig in!

**Comment by davidjharder** [comment]
Yes, almost every guide and doc will assume you are using the cli on a cloned local repo, pushing to some remote like Github.

Many modern editors are git-aware out of the box. For example, if you use vscode, it has a "source control" tab that will do simple git operations for you.

I use the Github editing tools for small one-off edits. As soon as you are editing more than one file, or organizing commits in any way, you will have to get comfortable with the cli

**Comment by davidjharder** [comment]
I should add: If you are struggling with something you can always ask for help in the Packaging or Development chat.

File: docs/user/editions/budgie/configuration.md [hunk 1]
```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
 
 ## Desktop Icons
 
-With Budgie, start by opening Raven, either by clicking on its icon in the far right of the system tray or using the keyboard shortcut `Super + A`. Next, click the cog icon for Budgie Settings.
+Desktop Icons are enabled by default when Budgie is installed for the first time. To check or toggle this setting, open the Budgie Desktop Settings app by selecting "Budgie Desktop Settings" from the Budgie Menu, or from the pop-up menu when right-clicking on a blank spot on the desktop.
```
Comment (davidjharder) [inline docs/user/editions/budgie/configuration.md:244]: "..enabled by default when Budgie is installed for the first time." is redundant. I would suggest something like:
Desktop Icons are enabled by default. To disable them, open...

File: docs/user/editions/budgie/configuration.md [hunk 2]
```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
...
-You may now add things to your Desktop. This can be done by dragging an item onto the Desktop or by right clicking an icon and selecting Copy to / Move to Desktop. Alternatively, open the Budgie Menu and click on "Files" and then move the files you want to the Desktop folder (as shown in the image below). Please note, the Desktop folder is not visible in "Files" until you've enabled the Desktop icons.
+To place files on your desktop, you can either drag them directly onto the Desktop, or move the files you want to the Desktop folder within the Nemo file manager app (as shown in the image below).
 
-![Budgie Desktop Folder](configuration/budgie-nautilus-desktop-folder.jpg)
+![Budgie Desktop Folder](configuration/budgie-nemo-desktop-folder.jpg)
 
 ## Workspaces
 
-Budgie enables the switching of workspaces via the Workspace Switcher. You can enable workspaces in Budgie by adding the Workspace Switcher applet to Raven.
+Budgie enables the switching of workspaces via the Workspace Switcher. You can enable workspaces in Budgie by adding the Workspace Switcher applet to your panel.
```
Comment (davidjharder) [inline docs/user/editions/budgie/configuration.md:256]: via -> with 
"with" is simpler

File: docs/user/editions/budgie/configuration.md [hunk 3]
```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
...
 
-Click the Raven settings 'cog' icon
+![Budgie Desktop Settings Panel Section](configuration/budgie-desktop-settings-bottom-panel.jpg)
 
-![Raven Settings Icon](configuration/raven-settings-cog.jpg)
-
-Alternatively, you can open the Budgie Menu and then click "Bottom Panel" in the System Settings category.
-
-![Budgie Panel Menu](configuration/budgie-panel-menu.jpg)
+Click the + button next to 'Add applet' on the right side of the window. This will open the list of available applets - scroll down until you find `Workspace Switcher`, select it, and then click the 'Add' button in the top-right of the window.
```
Comment (davidjharder) [inline docs/user/editions/budgie/configuration.md:262]: use quotes around "Workspace Switcher".
Backticks are reserved for package names, code, text entry, and so on

File: docs/user/editions/budgie/configuration.md [hunk 4]
```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
...
-
-![Budgie Workspaces](configuration/budgie-workspaces.jpg)
+![Budgie Workspaces](configuration/budgie-panel-with-workspace-switcher.jpg)
 
 ### Keyboard Shortcuts
 
 Budgie provides a variety of keyboard shortcuts for workspaces.
 
-To view these shortcuts open the Keyboard application and scroll to the Navigation section.
+To view these shortcuts, open the Budgie Control Center app, choose the Keyboard section in the left-hand sidebar, then scroll down in the right part of the window to choose 'Keyboard Shortcuts'.
```
Comment (davidjharder) [inline docs/user/editions/budgie/configuration.md:274]: Add quotes around "Keyboard" and change single quotes to double quotes around "Keyboard Shortcuts"

File: docs/user/editions/budgie/configuration.md [hunk 5]
```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
...
+
+![Budgie Control Center Keyboard Shortcuts](configuration/budgie-control-center-keyboard-shortcuts-location.jpg)
+
+Workspace shortcuts are contained in the 'Navigation' section of the Keyboard Shortcuts window.
```
Comment (davidjharder) [inline docs/user/editions/budgie/configuration.md:278]: Single to double quotes again

File: docs/user/editions/budgie/configuration.md [hunk 6]
```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
...
 
-![Budgie Workspaces Keyboard Shortcuts](configuration/budgie-workspace-shortcuts-gcc.jpg)
+![Budgie Control Center Keyboard Shortcuts Sections](configuration/budgie-control-center-keyboard-shortcuts-popup.jpg)
 
 ### Workspaces Configuration
 
-There are 4 workspaces available in Budgie by default. For adding an new Workspace, click on the `+` button.
+There are 4 workspaces available in Budgie by default. To add a new workspace, click on the `+` button.
```
Comment (davidjharder) [inline docs/user/editions/budgie/configuration.md:284]: For consistency, change the backticks to double quotes. Even though the backticked version kind of looks like a button

**Review by davidjharder — CHANGES_REQUESTED** [review]

**Comment by johnandmegh** [comment]
Thanks! I think I addressed the wording/consistency issues that you mentioned, and I scanned for similar quotation mark issues in at least the portion included in this request - I'll check the other one I sent as well.

**Review by davidjharder — APPROVED** [review]
Looks good.

**Pull request merged**